static/app/views/issueDetails/groupEventHeader.tsx

```diff
@@ -22,25 +20,11 @@ type GroupEventHeaderProps = {
 function GroupEventHeader({event, group, project}: GroupEventHeaderProps) {
   const organization = useOrganization();
   const issueTypeConfig = getConfigForIssueType(group, group.project);
-  const isRelatedIssuesEnabled = organization.features.includes(
-    'related-issues-issue-details-page'
-  );
-  // This is also called within the TraceTimeline component but caching will save a second call
-  const {isLoading, oneOtherIssueEvent} = useTraceTimelineEvents({
-    event,
-  });
-  const showTraceLink = isRelatedIssuesEnabled && !isLoading;
 
   return (
     <StyledDataSection>
       <GroupEventCarousel group={group} event={event} projectSlug={project.slug} />
-      {/* XXX: Remove TraceLink from groupEventCarousel when we GA. */}
-      {showTraceLink && (
-        <StyledTraceLink>
-          {oneOtherIssueEvent && <span>One other issue appears in the same trace.</span>}
-          <TraceLink event={event} />
-        </StyledTraceLink>
-      )}
+      {/* This renders either the trace timeline or the trace-related issue */}
       {issueTypeConfig.traceTimeline && <TraceTimeline event={event} />}
```

Comment:
If the `TraceTimeline` component can render itself conditionally based on whether the event has a trace id I think we can remove this configuration. The only issue type I'm aware of that it's disabled for is rage clicks which I don't think will ever have a trace id (and if they do, arguably it'd be nice to see the related issue/timeline anyway).

Reply:
Thanks for pointing that out. I will look a bit into it and report back:
```
git grep "issueTypeConfig.traceTimeline"
static/app/views/issueDetails/groupEventCarousel.tsx:        {issueTypeConfig.traceTimeline && <TraceLink event={event} />}
static/app/views/issueDetails/groupEventHeader.tsx:      {issueTypeConfig.traceTimeline && <TraceTimeline event={event} />}
```